mlir/lib/Dialect/ArmSME/Transforms/TileAllocation.cpp

```diff
@@ -289,6 +297,11 @@ struct LiveRange {
         .valid();
   }
 
+  /// Returns true if this range overlaps with `point`.
```

Comment:
What does `point` represent?

Reply:
It's simply a point in the program (every point in the program a number from `generateOperationNumbering()`).